libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -49,6 +49,7 @@ Implemented Papers
 - P2835R7: Expose ``std::atomic_ref``'s object address (`Github <https://llvm.org/PR118377>`__)
 - P2944R3: Comparisons for ``reference_wrapper`` (`Github <https://llvm.org/PR105424>`__)
 - P3168R2: Give ``std::optional`` Range Support (`Github <https://llvm.org/PR105430>`__)
+- P3836R2: Make ``optional<T&>`` trivially copyable (`Github <https://llvm.org/PR171275>`__)
```

Comment:
No change requested for this line.

Now we we should also update the entry in `Cxx2cPapers.csv`.
https://github.com/llvm/llvm-project/blob/b836942013978b3ab5d253da4569fc3ced7278b4/libcxx/docs/Status/Cxx2cPapers.csv#L165